.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: required
+language: python
+
+python:
+  - "2.7"
+
+addons:
+  apt:
+    packages:
+      - python-dev
+      - python-numpy
+      - python-opencv
+
+before_script:
+  - pip install pytest
+  - pip install numpy
+  - pip install opencv-python
+  - pip install -r requirements.txt
+  - python origami-lib/test/server_textimageinput.py &
+  - sleep 1
+
+script:
+  - cd origami-lib/test/
+  - pytest
```

Comment:
Please shut the server after script

Reply:
How?

Reply:
Nevermind. I'll make that a different task

Reply:
Good for me for now